### ISSUE TITLE
Crashes in Application

### ISSUE THREAD

**Issue by leedave** [leedave/flipper-zero-cross-remote#10]
There are few situations where the app crashes. 

I want to fix them, but finding the error often requires the ability to reproduce it in a reliable manner. 
Currently if I encounter a crash that restarts flipper, I can just do the same thing again and it works perfectly. 

If you're struggling with such a crash, please drop a message here in the following manner (example of one fixed bug)

```
Steps to recreate error
1. Start app
2. Delete existing command chain
3. Try to run an existing command chain
4. flipper crashes
```

**Comment by LardTom** [comment]
For me the app crashes when I try to run a newly created chain.

1: Create a chain
2: Without exiting the app try to run the chain
3: crash

(After the crash happened it works fine, but it happens with every new chain. I have to restart the flipper or the app to get it working without crashes)

**Comment by leedave** [comment]
@LardTom 
Thank you for the input. Can you give some info on what you put in the chain? 

I see that crash exists, but have problems finding the weak spot different tests give different outputs

1. IR -> ok
2. Pause -> ok
3. SubGhz -> ok
4. IR + Pause -> ok
5. SubGhz + Pause -> ok
6. IR + SubGhz -> ok
7. IR + Pause + SubGhz -> crash
8. SubGhz + Pause + IR -> crash

Not sure if it's related to the amount of entries or the combination of all 3. The error is always "null pointer dereference" which normally fixes well once you find the weak spot.

**Comment by leedave** [comment]
getting closer. It's not directly linked to running a newly created command. 

It actually happens when a newly created command, using multiple commands containing subghz is removed from buffer, to load a new one into it. The crash will happen in different approaches
- when trying to edit/delete any of the command chains
- when trying to run any command 

It also happens, when you don't save the newly created chain. So saving itself is not the issue

**Comment by leedave** [comment]
Found and fixed the issue with release 2.3

Also found a solution for the issue with the official app catalog. Should be added soon.
